docs/guides/migrating-from-camunda-7/data-migrator/limitations.md

```diff
@@ -173,6 +173,7 @@ The history migration has the following limitations.
 
 The [Cockpit plugin](/guides/migrating-from-camunda-7/data-migrator/cockpit-plugin.md) has the following limitations:
 
+- The migration schema has no authorization mechanism. Anyone with authenticated access to the Camunda 7 Cockpit can see the Cockpit Plugin and read the migration schema.
```

Comment:
Related to https://github.com/camunda/camunda-bpm-platform/issues/5427
This could be a warning, but that would potentially take focus from History or Migration limitations. It is clearly signaled that the Cockpit Plugin is experimental.